classes/i18n/LocaleConversion.php

```diff
@@ -198,4 +198,12 @@ public static function getIso1FromLocale(string $locale): ?string
         $localeMetadata = new LocaleMetadata($locale);
         return $localeMetadata->getIsoAlpha2();
     }
+
+    /**
+     * Convert Weblate locale to IETF BCP 47 language tag.
+     */
+    public static function toBcp47(string $locale): string
+    {
+        return str_replace(['_', '@'], '-', $locale);
```

Comment:
If you believe this can create problems somewhere, then:

```suggestion
    public static function toBcp47(?string $locale): string
    {
        return str_replace(['_', '@'], '-', $locale ?? '');
```

Reply:
I think it is OK to request the string. I already find out a few wrong things in our jatsTemplate tests -- an article and publication should always have 'locale'. Thus I will leave it as it is for now...